package.json

```diff
@@ -138,7 +138,7 @@
     "tslint": "5.14.0",
     "typescript": "^3.8.2",
     "typescript-to-proptypes": "^1.4.0",
-    "vrtest": "^0.2.0",
+    "vrtest-mui": "^0.3.3",
```

Comment:
I would actually inline it since it's only used in this repository. This way we can monitor changes directly without having to change, publish and update in this repository.

Reply:
It would definitely help to iterate on the logic. I couldn't run any of the tests of vrtest-mui. I can't only test the changes here.

Now, my hope would be that it's a one time only effort. I have no objection to inlining the dependency. My main concern is that I feel I could invest this time somewhere else with more ROI. I have no idea how much work it will be to inline the dependency. If we don't need to iterate for 12 months, I'm fine publishing 5 versions to iterate on the logic. At least for small changes, it's enough. For ambitious changes it would definitely be another story!

Reply:
If we need to change it again we'll reconsider inlining. Regarding tests: We   wouldn't need to run their test suite if we use it ourselves in CI. Basically just e2e testing.